tests/VM_with_planetary_network/planetary_network_test.go

```diff
@@ -27,12 +24,14 @@ func TestSingleNodeDeployment(t *testing.T) {
 
 	// retryable errors in terraform testing.
 	// generate ssh keys for test
-	tests.SshKeys()
-	publicKey := os.Getenv("PUBLICKEY")
+	pk, sk, err := tests.SshKeys()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
while I agree that there's no point to continue the tests without having the sshkeys geenrated, it should be moved into testing.M function instead and have it ready before everything